pkg/genericcli/printers/csv.go

```diff
@@ -0,0 +1,65 @@
+package printers
+
+import (
+	"bytes"
+	"encoding/csv"
+	"fmt"
+	"io"
+
+	"github.com/jszwec/csvutil"
+)
+
+type CSVPrinter struct {
+	c   *CSVPrinterConfig
+	out io.Writer
+}
+
+type CSVPrinterConfig struct {
+	// AutoHeader will generate headers during print, default is go standard ("false")
+	AutoHeader bool
+	// Delimiter the char to separate the columns, default is ";"
+	Delimiter rune
+}
+
+func NewCSVPrinter(config *CSVPrinterConfig) *CSVPrinter {
+	const defaultDelimiter = ';'
```

Comment:
const should be toplevel, not inside a func

Reply:
it's now toplevel